daxgen/daxgen.py

```diff
@@ -0,0 +1,184 @@
+import networkx as nx
```

Comment:
thanks for the info.   I'm sure you know this, but just in case:  if you intend to get that package in the final code you end up putting into the stack, you'll have to do an RFC and do the work to add that third party package before (or concurrently with) this packages code goes into the stack itself.